controllers/prep_handlers.go

```diff
@@ -287,7 +288,12 @@ func (r *ImageBasedUpgradeReconciler) SetupStateroot(ctx context.Context, ibu *l
 		return fmt.Errorf("failed to process etc.deletions: %w", err)
 	}
 
-	prep.BackupCertificates(ctx, osname, r.ManifestClient)
+	certsDir := common.PathOutsideChroot(
+		filepath.Join(common.GetStaterootPath(osname), "/var/opt/openshift/certs"),
```

Comment:
nit: We should aim to have all paths/filenames like this in consts. Maybe a separate PR for cleanup, to look for others that we could move to consts.